src/lib/chat/matrix-client.ts

```diff
@@ -518,6 +541,16 @@ export class MatrixClient implements IChatClient {
   };
 
   private publishMessageEvent(event) {
+    const room = this.matrix.getRoom(event.room_id);
+
+    if (!room) {
+      return;
+    }
+
+    if (event.sender !== this.userId) {
+      this.events.receiveUnreadCount(event.room_id, room.getUnreadNotificationCount(NotificationCountType.Total));
```

Comment:
Note : think this might have to move

Reply:
Yeah, this doesn't seem like the right place for this stuff. Receiving a message has nothing to do with unread counts.